client/js/lounge.js

```diff
@@ -178,6 +178,7 @@ $(function() {
 		}
 
 		$("body").removeClass("signed-out");
+		$("#loading").detach();
```

Comment:
This should have been `.remove`, we don't be reusing the element at a later time.